scripts/deploy-sdks.ts

```diff
@@ -276,7 +282,7 @@ function incrementVersion(version: string, command: string) {
 }
 
 function removeDevSuffix(version: string) {
-  return version.endsWith('-dev') ? version.slice(0, 5) : version
+  return version.endsWith('-dev') ? version.slice(0, -4) : version
```

Comment:
hmm why -4 here? is  from the end safer than from the start?

Reply:
Yes this was a mistake to have it like this. Two examples of this not working - 1.23.45 and 1.4.0-beta. This is why we erroneously released 1.4.0 instead of 1.4.0-beta.